runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -12,23 +12,23 @@ LanguageTree.__index = LanguageTree
 -- @param source Can be a bufnr or a string of text to parse
 -- @param lang The language this tree represents
 -- @param opts Options table
--- @param opts.queries A table of language to injection query strings.
---                     This is useful for overriding the built-in runtime file
---                     searching for the injection language query per language.
+-- @param opts.injection_queries A table of language to injection query strings.
+--                               This is useful for overriding the built-in runtime file
+--                               searching for the injection language query per language.
 function LanguageTree.new(source, lang, opts)
   language.require_language(lang)
   opts = opts or {}
 
-  local custom_queries = opts.queries or {}
+  local injection_queries = opts.injection_queries or {}
   local self = setmetatable({
     _source = source,
     _lang = lang,
     _children = {},
     _regions = {},
     _trees = {},
     _opts = opts,
-    _injection_query = custom_queries[lang]
-      and query.parse_query(lang, custom_queries[lang])
+    _injection_query = injection_queries[lang]
+      and query.parse_query(lang, injection_queries[lang])
       or query.get_query(lang, "injections"),
```

Comment:
only weird thing now is that we have `injection_queries` (as the key) and `injections` (as the file) which I'm not sure if that will be confusing.

Do you have any thoughts about that?

Reply:
changed!